spec/decorators/service_provider_session_decorator_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe ServiceProviderSessionDecorator do
+  subject { ServiceProviderSessionDecorator.new(sp_name: sp_name, sp_logo: nil) }
```

Comment:
I suggested the named subject, `subject(:decorator)` because then the tests can still read the same as they were before (`expect(decorator.foobar).to be_foobar`). I think un-named subjects are very easily confused, but again, small style nit, don't let that get in the way of this if you want to merge